Library/Formula/erlang.rb

```diff
@@ -47,6 +48,21 @@ class Erlang < Formula
 
   fails_with :llvm
 
+  if build.with? "timer-patches" and build.stable?
```

Comment:
If this only applies to stable, both the patches and the `option` definition should be in a `stable` block. We don't use `build.stable?` (or `head?` or `devel?`) in the DSL section anymore.